lib/megaco/doc/standard/draft-ietf-megaco-h248v2-04.txt

```diff
@@ -8903,7 +8903,7 @@
                                        Ref.: ITU-T Q.2931 
 
       SSCS        See      See table  See table in C.7 
-                  table in in C.7 
+                  table in C.7
```

Comment:
This is a standards document, we should not fix anything in it, be it typos or grammar.